### DISCUSSION TITLE
Ignore Format Changes in Diff

### DISCUSSION THREAD

**Discussion by PaulDMooney** [community/community#20610] [Pull Requests]
Similar to the "Ignore Whitespace" option to ignore whitespace changes in the diff view, it would be great to see a more enhanced version of that which ignores formatting changes.

For example if I had this:
```java
public static void myMethodIsTooLong(String becauseThis, String andThis, String andThis2) {
```
and it was reformatted to this:
```java
public static void myMethodIsTooLong(
    String becauseThis, String andThis, String andThis2) {
```
While nice to know we're applying a formatting tool to the code, I may not be interested in such a change and it becomes noise against the actual functional changes being made.

Granted this may have to be language specific as changes like this may carry functional differences in some languages (like python)

I believe this may also be mentioned in https://github.com/github-community/community/discussions/11479 but I am calling out this one functionality specifically.

**Comment by VersenyiZsolt** [comment]
Hey Paul,

Im currently developing a solution to these kind of problems. You can try it, maybe it works for you.

[www.archdiff.com](www.archdiff.com) 

Let me know your thoughts :)
